e2e-polybft/e2e/bridge_test.go

```diff
@@ -1442,3 +1442,112 @@ func TestE2E_Bridge_NonMintableERC20Token_WithPremine(t *testing.T) {
 		}
 	})
 }
+
+func TestE2E_Bridge_NonNative(t *testing.T) {
+	const (
+		transfersCount       = 5
+		amount               = 100
+		epochSize            = 5
+		numberOfAttempts     = 4
+		stateSyncedLogsCount = 2
+	)
+
+	var (
+		stakeAmount = ethgo.Ether(500)
+	)
+
+	minter, err := wallet.GenerateKey()
+	require.NoError(t, err)
+
+	receiverKeys := make([]string, transfersCount)
```

Comment:
Why do you need these transfers, receivers, amounts? You are not using this anywhere/